xml_converter/integration_tests/test_cases/xml_split_by_category/input/pack/xml_file.xml

```diff
@@ -0,0 +1,13 @@
+<OverlayData>
+    <MarkerCategory DisplayName="TopLevel1" Name="toplevel1" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" ></MarkerCategory>
+
+    <MarkerCategory DisplayName="TopLevel2" Name="toplevel2" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" ></MarkerCategory>
+
+    <MarkerCategory DisplayName="TopLevel3" Name="toplevel3" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" ></MarkerCategory>
```

Comment:
Lets differentiate these by more than just `DisplayName` and `Name`. Otherwise this test does a silly job at not catching the issue of "we write the same top level node but with different display/names for each category". The attributes could also be given to the POIs instead of the categories because XPos YPos ZPos dont make a huge amount of sense as category attributes.

Lets also add some subcategories here too so that we can test that "more than just the top level category gets written". Because we do want the entire category tree.